perun-utils/rpc-methods-javadoc-generator/parseRpcMethods.pl

```diff
@@ -723,7 +723,7 @@ sub processFile {
 
 		var url = document.location.toString();
 		if ( url.match('#') ) {
-			$('#'+url.split('#')[1]).addClass('in');
+			\$('#'+url.split('#')[1]).addClass('in');
```

Comment:
This is actually correct. \$ is necessary, since its JS inside perl. But the change is on master already (done by me), so I'm not sure, why github shows it as change for this pull-request.